In [1]:
import pandas as pd
import seaborn as sns

In [27]:
data=pd.read_csv('AI.csv', sep=',')

In [28]:
data

Timestamp  Age                                    Job  \
0    2020/02/25 3:28:17 PM GMT+6   19                                Студент   
1    2020/02/25 3:31:30 PM GMT+6   20                            Студент СДУ   
2    2020/02/25 3:32:48 PM GMT+6   19                                Студент   
3    2020/02/25 3:36:01 PM GMT+6   20                      И учусь и работаю   
4    2020/02/25 3:37:38 PM GMT+6   19                                  Учусь   
5    2020/02/25 3:38:01 PM GMT+6   20                     Студент маркетолог   
6    2020/02/25 3:43:29 PM GMT+6   19                      Маркетолог учусь    
7    2020/02/25 3:50:25 PM GMT+6   19                             Официантом   
8    2020/02/25 4:39:13 PM GMT+6   19                                SDUdent   
9    2020/02/25 5:15:39 PM GMT+6   19                                 Учусь    
10   2020/02/25 5:18:40 PM GMT+6   29                              Психолог    
11   2020/02/25 5:20:00 PM GMT+6   29                              Психолог    
12   2020/02/25 8:25:05 PM GMT+6   20                      Инженер-механиком   
13  2020/02/26 10:35:15 AM GMT+6   20                                студент   
14  2020/02/26 10:35:50 AM GMT+6   19                                  Учусь   
15   2020/02/26 3:16:38 PM GMT+6   19                                  Учусь   
16   2020/02/27 1:19:01 PM GMT+6   20                                студент   
17   2020/02/27 1:20:54 PM GMT+6   19                                Студент   
18   2020/02/27 1:21:23 PM GMT+6   19                                студент   
19   2020/02/27 1:21:51 PM GMT+6   18                                student   
20   2020/02/27 1:22:25 PM GMT+6   18                                  учусь   
21   2020/02/27 1:22:55 PM GMT+6   17                                  учусь   
22   2020/02/27 1:23:19 PM GMT+6   18                                   учус   
23   2020/02/27 1:23:42 PM GMT+6   19                                студент   
24   2020/02/27 1:24:16 PM GMT+6   17                                 учусьь   
25   2020/02/27 1:24:39 PM GMT+6   20                                студент   
26   2020/02/27 1:25:03 PM GMT+6   19                                  учусь   
27   2020/02/28 9:08:54 AM GMT+6   18                      Учусь на педагога   
28   2020/02/28 8:51:53 PM GMT+6   21                                  Учусь   
29   2020/02/28 8:53:08 PM GMT+6   53                            Домохозяйка   
..                           ...  ...                                    ...   
67  2020/02/28 11:30:04 PM GMT+6   19                               студент    
68  2020/02/28 11:30:05 PM GMT+6   18                               Учитель    
69  2020/02/28 11:30:35 PM GMT+6   19                             Студентка    
70  2020/02/28 11:30:42 PM GMT+6   20                               оператор   
71  2020/02/28 11:30:56 PM GMT+6   17                                 Учусь    
72  2020/02/28 11:31:41 PM GMT+6   15                                  учусь   
73  2020/02/28 11:33:00 PM GMT+6    5                                Student   
74  2020/02/28 11:33:34 PM GMT+6   19                                  Учусь   
75  2020/02/28 11:38:18 PM GMT+6   20                                 Геолог   
76  2020/02/28 11:38:23 PM GMT+6   19                                Студент   
77  2020/02/28 11:38:28 PM GMT+6   21                              медсестра   
78  2020/02/28 11:38:36 PM GMT+6   20                              Студентка   
79  2020/02/28 11:39:14 PM GMT+6   20                     проектный Менеджер   
80  2020/02/28 11:40:28 PM GMT+6   20                       Предприниматель    
81  2020/02/28 11:42:08 PM GMT+6   18                                  Учусь   
82  2020/02/28 11:42:25 PM GMT+6   20                        Учусь на Юриста   
83  2020/02/28 11:42:51 PM GMT+6   19                                 Учусь    
84  2020/02/28 11:45:16 PM GMT+6   20                             Не работаю   
85  2020/02/28 11:45:17

Input variables

* Timestamp is the time when form was filled

* Age is shows how old is a human

* Job is shows the workplace of the people

* Marital status is defining is human married or not, or divorced

* Monthly balance is the monthly salary of people

* Credit is shows if someone has credit or not

* Deposit means if someone has deposit or not

* Wanna deposit? defining if someone don't has deposit, he/she wants to deposit money in the future

In [29]:
data.head()

Timestamp  Age                Job        marital status  \
0  2020/02/25 3:28:17 PM GMT+6   19            Студент  Не женат(не замужем)   
1  2020/02/25 3:31:30 PM GMT+6   20        Студент СДУ  Не женат(не замужем)   
2  2020/02/25 3:32:48 PM GMT+6   19            Студент  Не женат(не замужем)   
3  2020/02/25 3:36:01 PM GMT+6   20  И учусь и работаю        Женат(замужем)   
4  2020/02/25 3:37:38 PM GMT+6   19              Учусь  Не женат(не замужем)   

   Monthly balance Credit Deposit Wanna deposit?  
0              0.0    Нет     Нет            Нет  
1              0.0    Нет     Нет            Нет  
2         100000.0    Нет     Нет            Нет  
3          70000.0    Нет      Да             Да  
4             45.0    Нет      Да            Нет

In [30]:
data.tail()

Timestamp  Age                                    Job  \
92  2020/02/28 11:56:51 PM GMT+6   20                                Студент   
93  2020/02/28 11:57:57 PM GMT+6   12                                 учусь    
94  2020/02/28 11:58:42 PM GMT+6   19                               студент    
95  2020/02/29 12:00:51 AM GMT+6   19  Студент, специальность: учёт и аудит    
96  2020/02/29 12:10:49 AM GMT+6   21                             Ресепшнист   

          marital status  Monthly balance Credit Deposit Wanna deposit?  
92  Не женат(не замужем)              0.0    Нет      Да             Да  
93  Не женат(не замужем)              0.0    Нет     Нет            Нет  
94  Не женат(не замужем)          30000.0    Нет      Да             Да  
95  Не женат(не замужем)              0.0    Нет     Нет             Да  
96  Не женат(не замужем)         100000.0    Нет     Нет            Нет

In [31]:
data.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 97 entries, 0 to 96
Data columns (total 8 columns):
Timestamp          97 non-null object
Age                97 non-null int64
Job                97 non-null object
marital status     97 non-null object
Monthly balance    97 non-null float64
Credit             97 non-null object
Deposit            97 non-null object
Wanna deposit?     91 non-null object
dtypes: float64(1), int64(1), object(6)
memory usage: 6.1+ KB


DataFrame has 97 entries (0 to 96) and 8 columns
* Age is int(number) variable
* Monthly balance is float(number) variable
* Timestamp, job, marital status, credit, deposit, wanna deposit? is objects(string variables)
* 1 int, 1 float, 6 string

In [32]:
data.shape

(97, 8)

In [33]:
data.describe()

Age  Monthly balance
count  97.000000     9.700000e+01
mean   20.134021     1.730899e+05
std     5.849909     1.023345e+06
min     5.000000     0.000000e+00
25%    18.000000     0.000000e+00
50%    19.000000     2.000000e+04
75%    20.000000     9.500000e+04
max    53.000000     1.000000e+07

* count means total emount of variables
* count of age is 97
* count of monthly balance is 97

* mean is adding all variables and deviding by their amount
* mean of age is 20.134021
* mean of monthly balance is 17308.99

* std is standard deviation of columns. The formula is: is the square root of the Variance, while Variance defined as the average of the squared differences from the Mean.
* std of age is 5.849909
* std of monthly balance is 10233.45

* min is the minimum variable
* min of age is 5
* min of monthly balance is 0

* max is the maximum variable in the column
* max of age is 53
* max of monthly balance is 10000000

* 25% shows the variable in the 25th percentale column, and respectively with 50% and 75%.


In [34]:
print(data.corr())

                      Age  Monthly balance
Age              1.000000        -0.233711
Monthly balance -0.233711         1.000000


* Table of correlation shows the relationship and connection between two variables in the table
* Age and monthly balance not releated to each other, consequently their correlation is -0.233711

In [35]:
print(data.cov())

                          Age  Monthly balance
Age              3.422143e+01    -1.399107e+06
Monthly balance -1.399107e+06     1.047236e+12


* Covariance is a measure of how much two random variables vary together.